include/test/battle.h

```diff
@@ -872,6 +903,14 @@ struct HPEventContext
     bool8 explicitCaptureDamage;
 };
 
+struct ExpEventContext
+{
+    u8 _;
+    u32 exp;
```

Comment:
```suggestion
```
I don't think `exp` is used in `QueueExp` so isn't needed here (which means it's impossible to directly ask for a specific amount of exp, although you could always capture it and then do a comparison with the captured value).

If you did want `exp`, then I think you probably need a `bool8 explicitExp;` after it, otherwise `APPEND_TRUE` will try to assign `TRUE` to `captureGainedExp` if we write `EXPERIENCE_BAR(exp: ...);`.